src/app/file-manager/file-explorer/file-explorer.component.html

```diff
@@ -81,6 +81,15 @@ <h1>Select files to
     </div>
   </div>
 
+  <div *ngIf="exportInProgress" class="warn-message">
```

Comment:
There should be a way to avoid the duplicate markup here. But that can wait for a future PR.